tldw_Server_API/tests/Audio/test_stt_vnext_config_flags.py

```
@@ -0,0 +1,154 @@
+def test_config_py_exports_canonical_stt_vnext_section(monkeypatch) -> None:
+    parser = ConfigParser()
+    parser.add_section("STT-Settings")
+    parser.set("STT-Settings", "ws_control_v2_enabled", "true")
+    parser.set("STT-Settings", "paused_audio_queue_cap_seconds", "9")
+    parser.set("STT-Settings", "overflow_warning_interval_seconds", "11")
+    parser.set("STT-Settings", "transcript_diagnostics_enabled", "true")
+    parser.set("STT-Settings", "delete_audio_after_success", "false")
+    parser.set("STT-Settings", "audio_retention_hours", "24")
+    parser.set("STT-Settings", "redact_pii", "true")
+
+    def _fake_load_comprehensive_config():
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `_fake_load_comprehensive_config`

(ANN202)